k8s-operator/utils.go

```diff
@@ -32,18 +32,12 @@ type Records struct {
 // TailscaledConfigFileName returns a tailscaled config file name in
 // format expected by containerboot for the given CapVer.
 func TailscaledConfigFileName(cap tailcfg.CapabilityVersion) string {
-	if cap < 95 {
-		return "tailscaled"
-	}
 	return fmt.Sprintf("cap-%v.hujson", cap)
 }
 
 // CapVerFromFileName parses the capability version from a tailscaled
 // config file name previously generated by TailscaledConfigFileNameForCap.
 func CapVerFromFileName(name string) (tailcfg.CapabilityVersion, error) {
-	if name == "tailscaled" {
-		return 0, nil
-	}
```

Comment:
Could we keep this? I know the config file is not designed for users to create, but it does seem nice to have one mechanism to read an unversioned config file with a stable name.

Reply:
> is not designed for users to create

I don't think we have consensus on this

Makes sense to keep it in any case

Reply:
Thanks! 👍